bedrock/firefox/templates/firefox/hello/start-45.html

```diff
@@ -27,8 +27,8 @@
   <section>
     <h2>{{ _('Made for sharing the Web.') }}</h2>
     <p>{{ _('Welcome to the all new Firefox Hello. Now when you invite a friend to a session, Firefox Hello will automatically share any Web page you’re viewing.') }} {{ _('Plan. Shop. Decide. Together.') }}</p>
-    {{ high_res_img('firefox/hello/2016/quick.png', {'alt': '', 'width': '45%', 'height': 'auto', 'class': 'screenshot'}) }}
```

Comment:
Hmm, interesting. I did clear out the static directory and those images did load. Good thing you caught it though.